.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Release checklist
+about: Prepare for a new release of scpca-docs
+title: Prepare for scpca-docs release
+labels: release
+assignees: ''
+
+---
+
+## Steps for a new release of `scpca-docs`
+<!-- Please include any notes about what will be included in this release -->
+<!-- e.g., you might want to mention if the release is related to a specific portal feature, like addition of AnnData objects -->
+<!-- Feel free to update the title of this issue to reflect the contents of the release (e.g., Prepare for scpca-docs release: AnnData) -->
+
+### Preparing for the release
+
+- [ ] Are all of the issues planned for this release resolved? If there are any issues that are unresolved, mark this issue as blocked by those on ZenHub.
```

Comment:
I believe "changes downloadable files" is the rule of thumb we'll use for CHANGELOG entries ([ref](https://data-lab-knowledge.slab.com/posts/sc-pca-portal-change-log-er8svvzg#h4j2h-what-goes-in-the-changelog)). I know this is already in the issue template for the data release/handoff, but another reminder here might not hurt?

```suggestion
- [ ] Does this release require an update to the CHANGELOG because there will be changes to downloadable files? If so, please create an issue tracking the CHANGELOG update and mark it as blocking this issue.
- [ ] Are all other the issues planned for this release resolved? If any issues are unresolved, mark this issue as blocked by those on ZenHub.
```

Reply:
We should definitely add this! I just forgot about the changelog...